### PR TITLE
Grid plots

### PR DESCRIPTION
**Pull request by igendr1**
- Generate_data function that takes in the x and y limits and generates random numbers, our 'stars', sampled from a uniform distribution
- star_flux function that takes  in an array of luminosities and star distances that return the fluxes. As of now all stars are set to a luminosity of 1.
- make_grid that plots in 2D space the positions of the stars with respect to the pixels and the coordinate bounds

### CODE REVIEW

File: CodingLab03/surface_b_fluct.py [hunk 1]
```diff
@@ -1,7 +1,71 @@
+import random
+import matplotlib.pyplot as plt
+
 def main():
+   
+
+def Generate_data(xmax,ymax,N):
+    """
+    Randomily generates (x,y) coordinates for stars and their luminosity
```
Comment (nlefrancois6) [inline CodingLab03/surface_b_fluct.py:9]: Typo: Randomly

File: CodingLab03/surface_b_fluct.py [hunk 2]
```diff
@@ -1,7 +1,71 @@
...
+
 
 def make_histogram():
 
-def make_grid():
+#nx and ny are the number of pixels in x coord and in y coord
+#ex: nx = 3 we want the xlim to be split into three pixels
+def make_grid(x_list,y_list,xmax,ymax,numBins_x,numBins_y):
+    """
+    Makes the grid of where each stars fall in our grid
```
Comment (nlefrancois6) [inline CodingLab03/surface_b_fluct.py:49]: Typo: star (not plural)

**Comment by katiesavarc** [comment]
This is good and will allow us to use variable luminosities in the future. I like the orange stars, nice touch!

**Comment by nlefrancois6** [comment]
Functions look good and the input/output commenting is good!

**Pull request merged**